### PR TITLE
Example for mixed swift/objc

### PR DESCRIPTION
**Pull request by JuanFelippo**
Example for mixed swift/objc sources.

### CODE REVIEW

File: Example/BUILD.bazel [hunk 1]
```diff
@@ -0,0 +1,54 @@
+load("@build_bazel_rules_ios//rules:app.bzl", "ios_application")
```
Comment (segiddins) [inline Example/BUILD.bazel]: Please nest the example under a subdirectory specificallyPlease nest the example under a subdirectory specifically about mixed-language frameworks

File: .bazelrc [hunk 2]
```diff
@@ -1,3 +1,3 @@
 # We can't create a bzl_library for rules-swift because of its visibility,
 # so circumvent by not using the sandbox
-build --strategy=Stardoc=standalone
+build --strategy=Stardoc=standalone --apple_platform_type=ios
```
Comment (segiddins) [inline .bazelrc]: please put this on a separate line, with a comment about why it it necessary

File: Example/BUILD.bazel [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+        "TestApp/**/*.swift",
+    ]),
+    bundle_id = "com.example.mixed-app",
+    families = [
+        "iphone",
+    ],
+
+    deps = [":ModuleA"],
+    minimum_os_version = "12.0",
+)
```
Comment (segiddins) [inline Example/BUILD.bazel]: might need to run `buildifier`

**Comment by mattrobmattrob** [comment]
Please reopen/rebase if you're still interested in contributing this change to the repo!

**Pull request closed without merging**